elfo-core/src/actor.rs

```diff
@@ -105,6 +110,32 @@ pub enum ActorStatusKind {
     Failed,
 }
 
+impl ActorStatusKind {
+    pub const fn is_normal(&self) -> bool {
+        matches!(self, Self::Normal)
+    }
+
+    pub const fn is_initializing(&self) -> bool {
+        matches!(self, Self::Initializing)
+    }
+
+    pub const fn is_terminating(&self) -> bool {
+        matches!(self, Self::Terminating)
+    }
+
+    pub const fn is_terminated(&self) -> bool {
+        matches!(self, Self::Terminated)
+    }
+
+    pub const fn is_alarming(&self) -> bool {
+        matches!(self, Self::Alarming)
+    }
+
+    pub const fn is_failed(&self) -> bool {
+        matches!(self, Self::Failed)
+    }
+}
```

Comment:
It seems that `Actor::is_terminating()` and `Actor::is_initializing()` can be replaced now by using `ctx.status_kind().is_*`

Reply:
Sure, did that in [1070425](https://github.com/elfo-rs/elfo/pull/133/commits/1070425102a5c11c57e2f0617259ec6dbfb6c18c)